src/test/unit/service/listManipulation/hearingparty/CrimeListsService.test.ts

```diff
@@ -5,90 +5,19 @@ import { CrimeListsService } from '../../../../../main/service/listManipulation/
 
 const crimeListsService = new CrimeListsService();
 const rawCrownDailyData = fs.readFileSync(
-    path.resolve(__dirname, '../../../mocks/hearingparty/crownDailyList.json'),
```

Comment:
The whole file can be removed. Also the test file crimeListParty.json can also go